lib/fake_artist/table.ex

```diff
@@ -243,4 +283,37 @@ defmodule FakeArtist.Table do
       current_seat + 1
     end
   end
+
+  @spec everyone_has_voted(map()) :: boolean()
+  defp everyone_has_voted(state) do
+    game_master_vote = 1
+
+    Enum.count(state.players |> Map.values(), fn x -> x.voted_for == nil end) ==
+      0 + game_master_vote
+  end
+
+  @spec get_winner(map()) :: binary()
+  defp get_winner(players) do
+    # TODO: Figure out who should win ties and make sure the right people win ties.
+    map_of_counts =
+      Enum.reduce(players |> Map.values(), %{}, fn player, acc ->
+        Map.update(acc, player.voted_for, 1, &(&1 + 1))
+      end)
+
+    [{key, _count} | _rest] = map_of_counts |> Map.to_list() |> Enum.sort_by(fn {_, v} -> v end)
```

Comment:
similar here. Because Enum works on Maps directly, it's probably an anti-pattern to do something like `Map.to_somethingElse |> Enum.fn`

Reply:
Also, use [`Enum.max_by/2`](https://hexdocs.pm/elixir/Enum.html#max_by/3) here rather than `sort_by |> hd()`

Reply:
Ah, I like it.